.gitlab-ci.yml

```diff
@@ -208,6 +208,16 @@ pinpoint-check:
     - *yarn_install
     - make lint_country_dialing_codes
 
+brakeman-check
```

Comment:
I think we want to run this on every PR since it's doing static analysis of the code written, I think [this](https://github.com/18F/identity-idp/pull/8322/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R74-L77) is the one we want to move?

```
bundle exec bundler-audit check --update
```

Reply:
ah yes my bad, thanks!

Reply:
updated in 431382ab7002433663c839c68b7e94ed02ecce57